notebooks/official/migration/UJ8 Vertex SDK AutoML Text Sentiment Analysis.ipynb

```diff
@@ -3,6 +3,7 @@
     {
```

Comment:
<p>Heading needs to be sentence cased</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1157/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>